silkworm/silkrpc/core/estimate_gas_oracle_test.cpp

```diff
@@ -102,7 +102,7 @@ TEST_CASE("estimate gas") {
 
     SECTION("Call empty, always fails but success in last step") {
         ExecutionResult expect_result_ok{.error_code = evmc_status_code::EVMC_SUCCESS, .pre_check_error = std::nullopt};
-        ExecutionResult expect_result_fail{.error_code = pre_check_error, .pre_check_error = "intrisic gas"};
+        ExecutionResult expect_result_fail{.error_code = std::nullopt, .pre_check_error = "intrisic gas"};
```

Comment:
You can simplify this by omitting `.error_code = std::nullopt, `

Reply:
OK